examples/Rakefile

```diff
@@ -1,41 +1,111 @@
 require 'fastly_nsq'
 require 'fastly_nsq/rake_task'
 
-##
-# Both topic and channel will need to be passed into the rake task when
-# it is called.
+#-------------------------------------------------------------------------------
+# You are required to to set the topics in MessageProcessor.topics as an array:
 #
-# ex: `rake listen_task[my_topic,my_channel]`
-# topic will be 'my_topic' and channel will be 'my_channel'
+# class MessageProcessor
+#   def self.topics
+#     ['customer_created', 'customer_deleted']
+#   end
 #
-MessageQueue::RakeTask.new(:listen_task, [:topic, :channel])
-
-##
-# In Rails, you can include the environment like so:
-#
-MessageQueue::RakeTask.new(:listen_task, [:topic, :channel] => :environment)
-
-##
-# Topic and channel can also be preset in the task by passing a block and
-# assigning value like so. This is useful if you want a single purpose
-# listener task.
+#   # ...
+# end
```

Comment:
Have you considered uncommenting this as it would still be valid ruby and makes some things later make more sense when you reference what the topics will become.